doc/contributing/coding-conventions.chapter.md

```diff
@@ -196,7 +196,20 @@
   As an exception, an explicit conditional expression with null can be used when fixing a important bug without triggering a mass rebuild.
   If this is done a follow up pull request _should_ be created to change the code to `lib.optional(s)`.
 
-- Arguments should be listed in the order they are used, with the exception of `lib`, which always goes first.
+- The order of elements in the argument set should follow the convention below:
+  1. `lib`;
+  2. `stdenv`;
+  3. builders (e.g. `buildPythonPackage`, `mkDerivation`, `buildGoModule`);
+  4. fetchers (e.g. `fetchurl`, `fetchpatch`, `fetchFromGitHub`);
+  5. all the remaining arguments (i.e. those that don't fit the other items);
+  6. options (e.g. `extraModules`);
+  7. Boolean flags (e.g. `enableSDL`);
+  8. `buildPackages`;
+  9. Darwin-related arguments (e.g. `darwin`, `IOKit`, `Metal`).
+
+  Each item is optional, in the sense it should not be included when not needed (e.g. if there is no need for `buildPackages`, then it should not be included).
+
+  Inside each item of the hierarchy above, the arguments should be in "ASCIIbetical" order, (e.g. `SDL` comes before `Xaw3d`, that comes before `alsa-lib`, that comes before `assemblyscript`).
```

Comment:
```suggestion
  Inside each item of the hierarchy above, the arguments should be in "ASCIIbetical" order.
Example:`SDL`, `Xaw3d`, `alsa-lib`, `assemblyscript`.
```

Just make a concrete example, avoid filling words that don't convey information - that it easier to scan. 

Is there a particular reason for this oddly specific convention though?